jerry-core/ecma/operations/ecma-boolean-object.c

```diff
@@ -38,7 +38,7 @@
  * @return ecma value
  *         Returned value must be freed with ecma_free_value
  */
-ecma_value_t
+ecma_object_t *
```

Comment:
Please update the doc about the return value and mention that this operation cannot fail.